packages/ui/src/layout/DashboardLayout.tsx

```diff
@@ -1,19 +1,41 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
This will still break mobile when the use clicks on the 3 bars icon. Can you please use the implementation which was already provided in the original dashboard layout? Otherwise just remove the new sidebar component and just change the style that you worked on straight into that component, whatever is easier.

Reply:
No it will not break the mobile view I have tested all devices 

Take a pull of that branch and have a look at the UI

Reply:
What I mean is that the sidebar feature was already done. No need to create a new one. If you want to split it, it's a good idea, but the original version was working good and had animation, which your implementation doesnt.

Reply:
Can you please do those changes you are asking for?

Reply:
Yeah sure. Thanks a lot for your PR. I will take care of the rest.

Reply:
Thank you

Reply:
Is there anything else you would like to work on?

Reply:
Sure, I can work on the UI pieces and API integrations.